andela/learning-map-front
# #150473192 Delete bookmarks from the sidebar

#### What does this PR do?
Enables the deleting of any bookmark from a list of bookmarks on the sidebar

#### Description of Task to be completed?
This PR is to complete the following tasks:
- Fix getting wrong token from the cookie
- Rename a misleading class method in upvoteSection
- Create a delete button with tooltip for each bookmark
- Implement the deleting of bookmarks via delete button on the sidebar
- Display a snackbar notification when a bookmark is deleted

#### How should this be manually tested?
- Clone the repository
- Move into the project directory
- Run yarn install
- Run yarn start
- Log in with your andela email
- When you're successfully logged in, you should see the different contributions
- Bookmark a contribution with the bookmark icon/button
- Open the sidebar to see list of bookmarked contributions
- Hover on the delete button to see the tooltip
- Click the delete button to see the bookmark removed from sidebar and that on the contribution list unbookmarked.

#### Any background context you want to provide?
There was no way of deleting bookmarks from the sidebar

#### What are the relevant pivotal tracker stories?
[#150473192](https://www.pivotaltracker.com/story/show/150473192)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/25608624/30376808-8a307776-9885-11e7-8f05-c0f753f68c43.png)
![image](https://user-images.githubusercontent.com/25608624/30376821-9183b880-9885-11e7-90c7-d78539800f6d.png)
![image](https://user-images.githubusercontent.com/25608624/30376826-982dc464-9885-11e7-8296-cc7e81f2a904.png)
![image](https://user-images.githubusercontent.com/25608624/30376832-9c13e220-9885-11e7-8bb9-8393f64f3c9d.png)

#### Questions:
N/A
